core/src/main/java/com/linecorp/armeria/server/logging/AccessLogger.java

```diff
@@ -100,15 +102,14 @@
 final class AccessLogger {
     private static final Logger logger = LoggerFactory.getLogger(AccessLogger.class);
 
-    private static final Logger accessLogger =
-            LoggerFactory.getLogger("com.linecorp.armeria.logging.access");
-
     /**
      * Writes an access log for the specified {@link RequestLog}.
      */
     static void write(List<AccessLogComponent> format, RequestLog log) {
+        final VirtualHost host = ((ServiceRequestContext) log.context()).virtualHost();
+        final Logger logger = host.accessLogger();
         if (!format.isEmpty()) {
```

Comment:
Not related to this pull request but could you also add a check for `logger.isInfoEnabled()`?